### DISCUSSION TITLE
Where can I find the full list of modules available?

### DISCUSSION THREAD

**Discussion by luiz00martins** [nvim-neorg/neorg#71] [Q&A] [answered]
Currently I'm getting an `Error executing lua .../lua/neorg/modules/core/integration/nvim-compe/module.lua:22: nvim-compe not found, aborting...`. This is most likely because `core.defaults` is loading `core.integrations.nvim-compe`, and I don't have `nvim-compe` installed.

So, I'd like to load the modules manually, so I can pick the ones I'll use. However, after searching through the repo and docs, I was not able to find a list of modules.

Is there an extensive list somewhere?

**Comment by vhyrro** [comment, marked as answer]
The wiki contains a list of *documented* modules but for the past month or so I've been focusing on the code and leaving docs until before the 0.1 release. I know that the completion errors are a bit annoying and i will be fixing it tomorrow and forcing users to select a completion engine manually. As of now you probably want to configure core.norg completion as such:

```lua
["core.norg.completion"] = {
    config = {
        engine = nil
    }
}
```

Or optionally you can disable the module entirely:
```lua
["core.defaults"] = {
    config = {
        disable = { "core.norg.completion" },
    }
}
```

Note that setting the engine to `nil` will cause warnings to show up, I'll be fixing that tomorrow too :)

Looking at lua/neorg/modules/core is also a good way of finding neorg modules (duh lol). Each one should have a short description at the top of the file about what it does. Hope that helps!

**Reply by luiz00martins** [reply]
Of course, the Wiki! Don't know how I forgot about that.

> Looking at lua/neorg/modules/core is also a good way of finding neorg modules (duh lol).

lmao, yeah, that's basically (well... literally) a list of all the modules.

In any case, I think that `disable = { "core.norg.completion" }` is a more elegant solution to my problem. Thanks for that!
